optimizely/optimizely_config.py

```diff
@@ -0,0 +1,213 @@
+# Copyright 2019, Optimizely
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+
+
+class OptimizelyConfig(object):
+    def __init__(self, revision, experiments_map, features_map):
+        self.revision = revision
+        self.experiments_map = experiments_map
+        self.features_map = features_map
+
+
+class OptimizelyExperiment(object):
+    def __init__(self, id, key, variations_map):
+        self.id = id
+        self.key = key
+        self.variations_map = variations_map
+
+
+class OptimizelyFeature(object):
+    def __init__(self, id, key, experiments_map, variables_map):
+        self.id = id
+        self.key = key
+        self.experiments_map = experiments_map
+        self.variables_map = variables_map
+
+
+class OptimizelyVariation(object):
+    def __init__(self, id, key, feature_enabled, variables_map):
+        self.id = id
+        self.key = key
+        self.feature_enabled = feature_enabled
+        self.variables_map = variables_map
+
+
+class OptimizelyVariable(object):
+    def __init__(self, id, key, type, value):
+        self.id = id
+        self.key = key
+        self.type = type
+        self.value = value
+
+
+class OptimizelyConfigService(object):
+    """ Class encapsulating methods to be used in creating instance of OptimizelyConfig. """
+
+    def __init__(self, project_config):
+        """
+        Arguments:
+            project_config ProjectConfig
+        """
+        self.experiments = project_config.experiments
+        self.feature_flags = project_config.feature_flags
+        self.groups = project_config.groups
+        self.revision = project_config.revision
+
+    def get_optimizely_config(self):
```

Comment:
nit. Not sure if the naming has to be `get_optimizely_config`, but `OptimizelyConfigService.get_optimizely_config` has too much redundant information in it.

Something as simple as `get_config` may suffice here. cc @jaeopt